src/Raven.Studio/typescript/viewmodels/database/tasks/editRavenEtlTask.ts

```diff
@@ -439,7 +439,7 @@ class editRavenEtlTask extends viewModelBase {
         
         if (transformation.isNew()) {
             const newTransformationItem = new ongoingTaskRavenEtlTransformationModel(transformation.toDto(), true, false); 
-            newTransformationItem.name(this.findNameForNewTransformation());
+            newTransformationItem.name(transformation.name() || this.findNameForNewTransformation());
```

Comment:
we should assign and display this name when user enters the view.

Reply:
done - showing "Script_1" immediately - upon opening view

Reply:
do we need that? At this point name should be valid (not empty)

Reply:
ok - removed